api/src/pcapi/core/users/email/update.py

```diff
@@ -60,6 +60,10 @@ def generate_and_send_beneficiary_confirmation_email_for_email_change(user: mode
     expiration_date = generate_email_change_token_expiration_date()
 
     token = generate_email_change_token(user, new_email, expiration_date, TokenType.CONFIRMATION)
+    check_email_address_does_not_exist(
+        new_email
+    )  # We want to execute everything normally in this case except for sending the email to avoid giving away if the email exists or not
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Je pense pas que le commentaire apporte grand chose ici
C'est déjà expliqué quand on catch l'exception dans la route